nvflare/apis/impl/wf_comm_client.py

```diff
@@ -220,21 +239,49 @@ def send(
         targets: Union[List[Client], List[str], None] = None,
         send_order: SendOrder = SendOrder.SEQUENTIAL,
         task_assignment_timeout: int = 0,
+    ):
+        return self.send_and_wait(task, fl_ctx, targets, send_order, task_assignment_timeout)
+
+    def send_and_wait(
+        self,
+        task: Task,
+        fl_ctx: FLContext,
+        targets: Union[List[Client], List[str], None] = None,
+        send_order: SendOrder = SendOrder.SEQUENTIAL,
+        task_assignment_timeout: int = 0,
+        abort_signal: Signal = None,
     ):
         engine = fl_ctx.get_engine()
 
         self._validate_target(engine, targets)
 
-        return self.send_and_wait(task, fl_ctx, targets, send_order, task_assignment_timeout)
+        for target in targets:
+            reply = self.broadcast_and_wait(task, fl_ctx, [target], abort_signal=abort_signal)
+            if reply.get_return_code() == ReturnCode.OK:
+                return reply
```

Comment:
So our `send_and_wait` is expected to be 
"send 1st -> 1st execution -> recv 1st -> send 2nd -> 2nd execution -> recv 2nd"?

Or we can do these in parallel:

"send 1st -> 1st execution -> recv 1st"
"send 2nd -> 2nd execution -> recv 2nd"
"send 3rd -> 3rd execution -> recv 3rd"

Reply:
`send_and_wait()` was originally defined in our controller as sending a task to a single target, wherein the list of targets is just a list of priority of backup targets in case sending fails. The parallel version is essentially `broadcast()`, and the sequential version would be `relay()`.

Reply:
Does this mean that our `send_and_wait` now for multiple targets is almost equivalent to `relay_and_wait` with task_assignment_timeout set to 0?

Reply:
The difference is that `send_and_wait` will finish after sending the task to one successful target, while `relay_and_wait` will send the task to all of the targets in sequential order.

Reply:
got it, sounds good, this is just following the original behavior (although that is a bit tricky to get around).
But for the scope of this PR, this is good.